certbot/main.py

```diff
@@ -275,7 +277,8 @@ def _ask_user_to_confirm_new_names(config, new_domains, certname, old_domains):
                new_domains,
                old_domains))
     obj = zope.component.getUtility(interfaces.IDisplay)
-    if not obj.yesno(msg, "Update cert", "Cancel", default=True):
+    if not obj.yesno(msg, "Update cert", "Cancel",
```

Comment:
You're right! I thought I excluded all of the new prompts in previously existing code paths but looks like I added it to this one by mistake. Thanks for catching it.